### PR TITLE
chore: Bypass caching across all environments

### PR DESCRIPTION
**Pull request by nlewis84**
## 🐛 Issue

Sometimes you are making changes in Admin and need to see in Web Embeds

## ✏️ Solution

Add documentation for how to disable caching

## 🔬 To Test

1. Start Admin and Web Embeds
2. Uncomment the `x-cache-me-not` header
3. Change a content item on Apollos Demo (on Admin) and nav to that same item in web embeds (or micro-service)
4. See the change come through immediately

## 📸 Screenshots

https://github.com/ApollosProject/apollos-embeds/assets/72768221/0d41f7e2-af35-42d4-8f1d-b04e29ee06ec

### CODE REVIEW

**Comment by vinnyjth** [comment]
@nlewis84 I love this but let's turn caching off by default :D

File: packages/web-shared/client/apollosApiLink.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ const apollosApiLink = (church_slug) =>
       headers: {
         ...headers,
         'x-church': church_slug,
+        // Uncomment the following line to disable caching
+        // 'x-cache-me-not': 1,
```
Comment (vinnyjth) [inline packages/web-shared/client/apollosApiLink.js:12]:
```suggestion
        'x-cache-me-not': 1,
```

File: micro-service/README.md [hunk 2]
```diff
@@ -66,6 +66,18 @@ For the 'FeatureFeed' embed, which displays the church's content, add `data-type
 
 _⚠️ Make sure to replace [INSERT_CHURCH_SLUG_HERE] with your church's unique identifier, or 'slug'._
 
+### Disabling Caching for Local Testing
+
+For local development and testing purposes, you might want to disable caching to ensure you're receiving the latest responses directly from the API. To do this, please refer to the Apollo client configuration file:
```
Comment (vinnyjth) [inline micro-service/README.md:71]:
```suggestion
For local development and testing purposes, you might want to enable caching to ensure you're not receiving the latest responses directly from the API. To do this, please refer to the Apollo client configuration file:
```

File: micro-service/README.md [hunk 3]
```diff
@@ -66,6 +66,18 @@ For the 'FeatureFeed' embed, which displays the church's content, add `data-type
...
+
+[../packages/web-shared/client/apollosApiLink.js](../packages/web-shared/client/apollosApiLink.js)
+
+In this file, locate the header configuration within the `apollosApiLink` function and uncomment the following line:
```
Comment (vinnyjth) [inline micro-service/README.md:75]:
```suggestion
In this file, locate the header configuration within the `apollosApiLink` function and comment the following line:
```

File: micro-service/README.md [hunk 4]
```diff
@@ -66,6 +66,18 @@ For the 'FeatureFeed' embed, which displays the church's content, add `data-type
...
+
+```javascript
+// 'x-cache-me-not': 1,
```
Comment (vinnyjth) [inline micro-service/README.md:78]:
```suggestion
'x-cache-me-not': 1,
```

File: web-embeds/README.md [hunk 5]
```diff
@@ -76,6 +76,18 @@ For the 'FeatureFeed' embed, which displays the church's content, add `data-type
...
+### Disabling Caching for Local Testing
+
+For local development and testing purposes, you might want to disable caching to ensure you're receiving the latest responses directly from the API. To do this, please refer to the Apollo client configuration file:
+
+[../packages/web-shared/client/apollosApiLink.js](../packages/web-shared/client/apollosApiLink.js)
+
+In this file, locate the header configuration within the `apollosApiLink` function and uncomment the following line:
+
+```javascript
+// 'x-cache-me-not': 1,
```
Comment (vinnyjth) [inline web-embeds/README.md:88]:
```suggestion
### Disabling Caching for Local Testing

For local development and testing purposes, you might want to enable caching to ensure you're not receiving the latest responses directly from the API. To do this, please refer to the Apollo client configuration file:

[../packages/web-shared/client/apollosApiLink.js](../packages/web-shared/client/apollosApiLink.js)

In this file, locate the header configuration within the `apollosApiLink` function and comment the following line:

```javascript
'x-cache-me-not': 1,
```

File: micro-service/README.md [hunk 6]
```diff
@@ -66,6 +66,18 @@ For the 'FeatureFeed' embed, which displays the church's content, add `data-type
...
+
+For local development and testing purposes, you might want to enable caching to ensure you're not receiving the latest responses directly from the API. To do this, please refer to the Apollo client configuration file:
+
+[../packages/web-shared/client/apollosApiLink.js](../packages/web-shared/client/apollosApiLink.js)
+
+In this file, locate the header configuration within the `apollosApiLink` function and comment the following line:
+
+```javascript
+'x-cache-me-not': 1,
+
```
Comment (nlewis84) [inline micro-service/README.md:79]:
```suggestion
```

```

**Comment by nlewis84** [comment]
@vinnyjth Got your changes in (and fixed a typo I saw)

**Review by vinnyjth — APPROVED** [review]
🙌

**Comment by vinnyjth** [comment]
updating the pr title to avoid future confusion ;)

**Pull request merged**